### PR TITLE
Variable Precision Manipulation System

### PR DESCRIPTION
**Pull request by Zee2**
## Overview
Precision manipulation widgets.

image

DRAFT PR

### CODE REVIEW

File: Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/BoundsControl.cs [hunk 1]
```diff
@@ -128,6 +159,8 @@ public BoundsControlActivationType BoundsControlActivation
             }
         }
 
+        [Header("Visuals")]
```
Comment (thalbern) [inline Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/BoundsControl.cs:162]: >[Header("Visuals")] [](start = 8, length = 19)

removed this as we're defining the headings anyways in the inspector

File: Assets/MRTK/Examples/Experimental/BoundsControl/Scenes/BoundsControlConfigs/Precision Configs/PrecisionRotationHandlesConfiguration.asset [hunk 2]
```diff
@@ -0,0 +1,29 @@
+%YAML 1.1
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Experimental/BoundsControl/Scenes/BoundsControlConfigs/Precision Configs/PrecisionRotationHandlesConfiguration.asset:1]: This is a SUPER-nit, but would it be possible to rename `Precision Configs` to `PrecisionConfigs`? We've tended to try to not have spaces in folder / file names.

**Comment by Zee2** [comment]
Closing draft PR to split into multiple PRs.

**Pull request closed without merging**